src/main/resources/static/index.html

```diff
@@ -88,24 +84,18 @@
           </div>
         </div>
         <div class="ui divider horizontal">Or</div>
-        <div class="field required">
-          <div class="ui left action input">
-            <button class="ui button icon teal" id="geocoding">
+        <div class="ui left action input" id="pac-card">
+            <div class="ui button icon teal">
               <i class="icon search"></i>
-            </button>
-            <div class="pac-card" id="pac-card">
-              <div id="pac-container">
-                <input id="pac-input" type="text"
-                       placeholder="Your city" name="place_input">
-                <input type="hidden" name="place_output">
-              </div>
-            </div>
-            <div id="infowindow-content">
-              <img src="" width="16" height="16" id="place-icon">
-              <span id="place-name" class="title"></span><br>
-              <span id="place-address"></span>
             </div>
-          </div>
+          <input id="pac-input" type="text"
+                 placeholder="Your workplace..." name="place_input">
```

Comment:
No need for "..."

Reply:
`name` can be `location` and you can add this to `Bee` to store it and display it again when we display the form on the next visit of the user.

Reply:
Done